src/Microsoft.AspNet.Http/Features/FormFile.cs

```diff
@@ -3,20 +3,23 @@
 
 using System.IO;
 using Microsoft.AspNet.Http.Internal;
+using Microsoft.Net.Http.Headers;
+using static Microsoft.Net.Http.Headers.HeaderUtilities;
 
 namespace Microsoft.AspNet.Http.Features.Internal
 {
     public class FormFile : IFormFile
     {
-        private Stream _baseStream;
-        private long _baseStreamOffset;
-        private long _length;
+        private readonly Stream _baseStream;
+        private readonly long _baseStreamOffset;
 
-        public FormFile(Stream baseStream, long baseStreamOffset, long length)
+        public FormFile(Stream baseStream, long baseStreamOffset, long length, ContentDispositionHeaderValue contentDisposition)
```

Comment:
Can we move the quote removal on to the outside and just pass the name and file name. It's weird to have the header dependency in here (always was)